.bmad-core/templates/architecture-tmpl.yaml

```
@@ -0,0 +1,650 @@
+  elicitation: advanced-elicitation
+
+sections:
+  - id: introduction
+    title: Introduction
+    instruction: |
+      If available, review any provided relevant documents to gather all relevant context before beginning. If at a minimum you cannot locate docs/prd.md ask the user what docs will provide the basis for the architecture.
+    sections:
+      - id: intro-content
+        content: |
+          This document outlines the overall project architecture for {{project_name}}, including backend systems, shared services, and non-UI specific concerns. Its primary goal is to serve as the guiding architectural blueprint for AI-driven development, ensuring consistency and adherence to chosen patterns and technologies.
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)